services/notifications/docs/Notifications.md

```diff
@@ -0,0 +1,150 @@
+Notifications
+======
+
+GET /notifications/
+---------------------
+
+Returns a list of all notification topics.
+
+Response format:
+```
+{
+    "topics": [
+        {
+            "arn": "arn:aws:sns:us-east-2:256758753660:Mentor",
```

Comment:
I don't think we should be exposing the arn of the topics to API users. API users should be interacting with the topics solely through the topic names.